db/models/patient.js

```diff
@@ -63,9 +63,10 @@ function setEmailAndPassword(patient) {
 }
 
 function randAlpha(num) {
-  let rand = '', idx=0, chars = 'abcdefghijklmnopqrstuwxyzABCDEFGHIJKLMNOPQRSTUWXYZ0123456789'
+  let rand = '', idx=0;
+  const chars = 'abcdefghijklmnopqrstuwxyzABCDEFGHIJKLMNOPQRSTUWXYZ0123456789'
 
-  while (++idx < num) {
+  while (idx++ < num) {
```

Comment:
Because this is a while loop, this tested fine for me with ++idx to produce _n_ digit random alphanumeric. If you'd prefer to change, maybe idx += 1